packages/core/src/builder.class.ts

```diff
@@ -563,6 +563,11 @@ export interface Component {
    * for more information on permissions in builder check https://www.builder.io/c/docs/guides/roles-and-permissions
    */
   requiredPermissions?: Array<Permission>;
+
+  /**
+   * Used to store additional component information.
+   */
+  meta?: Record<string, any>;
```

Comment:
Oh, can you add this to the `ComponentInfo` in the new SDKs as well? Your change currently only applied to the gen1 React SDK

https://github.com/BuilderIO/builder/blob/main/packages/sdks/src/types/components.ts#L4